platform/x11/power_x11.cpp

```diff
@@ -58,6 +58,7 @@ Adapted from corresponding SDL 2.0 code.
 #include <stdio.h>
 #include <unistd.h>
 
+#include <core/error_macros.h>
```

Comment:
Godot includes should use double quotes, `<>` are for system headers. So it should be:
```
#include "power_x11.h"

#include "core/error_macros.h"

#include <dirent.h>
 #include <fcntl.h>
#include <stdio.h>
#include <sys/stat.h>
#include <sys/types.h>
```
#include <unistd.h>

Reply:
Wasn't aware of this standard. I'll fix it then.